apps/aeoracle/src/aeo_state_tree.erl

```diff
@@ -31,26 +34,23 @@
 %% in the oracle field in the state tree, does not work well. Since the state
 %% tree has to be a Merkle tree all nodes are serialized. This would mean
 %% deserialize/serialize of all queries when adding or updating a single
-%% query. Instead we have a forest of Merkle trees (on Merkle tree per
-%% oracle) in a separate structure, and only store the root hash of that
-%% queries tree in the oracle field.
+%% query. Instead we store the queries prefixed with the oracle id in
+%% the same tree as the oracles. This is to enable iteration over a single
+%% oracle's queries.
```

Comment:
Requires update of https://github.com/aeternity/protocol/blob/da3a79f9399f73f78833bb74f51ddd66b49ac647/oracles/oracle_state_tree.md#oracle-state-tree